Simplenote/src/main/java/com/automattic/simplenote/NoteEditorFragment.java

```diff
@@ -542,13 +563,33 @@ public void onGlobalLayout() {
                     int lineTop = layout.getLineTop(layout.getLineForOffset(matchLocation));
                     ((NestedScrollView) mRootView).smoothScrollTo(0, lineTop);
                     mShouldScrollToSearchMatch = false;
+                } else if (mNote != null && mNote.getSimperiumKey() != null) {
+                    ((NestedScrollView) mRootView).scrollTo(0, mPreferences.getInt(mNote.getSimperiumKey(), 0));
+                    mRootView.setOnScrollChangeListener(
+                        new View.OnScrollChangeListener() {
+                            @Override
+                            public void onScrollChange(View v, int scrollX, int scrollY, int oldScrollX, int oldScrollY) {
```

Comment:
One issue here is that since the `scrollY` is being saved & restored, it won't be accurate if the note is re-opened with the device in a different orientation than when the position was saved.

Reply:
Just occurred to me that it also won't be accurate if the note is edited on another device.

Reply:
np: the cast to `(NestedScrollView)` isn't necessary since all views support `scrollTo`.

Reply:
Yeah, this isn't a perfect solution for all situations.  The use case is when someone is editing while moving back and forth between notes in a relatively short session.  Things like remote changes and device orientation cause the scroll position to be inaccurate at times.  This is meant to try to have some feature parity between platforms even though this specific feature might be optimal on other platforms.

Reply:
> Yeah, this isn't a perfect solution for all situations. The use case is when someone is editing while moving back and forth between notes in a relatively short session.

Makes total sense, I just think it could be annoying for someone who (like me) edits on multiple devices to be thrown to a random scroll position. Perhaps in a separate PR we could also store the number of characters along with the scroll position, and if different than when it was stored we default to not restoring the position. Likewise for orientation.